src/convertToObject.js

```diff
@@ -6,7 +6,36 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const splitThis = sourceString.split(';');
+  const result = {};
+
+  for (const oneString of splitThis) {
+    let word = '';
+    let wordTwo = '';
+    let fazeTwo = false;
```

Comment:
Variables `word` and `wordTwo` are not descriptive of their purpose. Use more descriptive names like `property` and `value` respectively. Also, the variable `fazeTwo` is a flag that indicates the parsing phase but is not named appropriately. Consider renaming it to `isParsingValue` for clarity.